OTAnalytics/application/progress.py

```diff
@@ -23,25 +23,35 @@ def __init__(
         sequence: Sequence,
         counter: Counter,
         notify: Optional[Callable[[], None]] = None,
+        step_percentage: int = 1,
     ) -> None:
         self._sequence = sequence
         self._counter = counter
         self._notify = notify
+        self._step_percentage = step_percentage * 100
```

Comment:
Why do we need to multiply the value by 100?

Reply:
I do not know anymore.